index.bs

```diff
@@ -4887,15 +4793,12 @@ partial interface MLGraphBuilder {
     1. Set |descriptor|.{{MLOperandDescriptor/dataType}} to |input|'s [=MLOperand/dataType=].
     1. Set |descriptor|.{{MLOperandDescriptor/dimensions}} to the result of [=unidirectionally broadcasting the shapes=] |slope|'s [=MLOperand/shape=] and |input|'s [=MLOperand/shape=].
         1. If that returns failure, then [=exception/throw=] a {{TypeError}}.
-    1. If any of the following sub-steps fail, [=exception/throw=] an "{{OperationError}}" {{DOMException}}.
+    1. *Make graph connections:*
         1. Let |output| be the result of [=creating an MLOperand=] given [=this=] and |descriptor|.
-        1. Make a request to the underlying platform to:
-            1. Let |opImpl| be [=platform operator=] for the PreLU operation, given |slope|.
-            1. Set |output|.{{MLOperand/[[operator]]}} to |opImpl|.
-            1. Create an [=platform operand=] |outputImpl| to represent the output, given |output| and |opImpl|.
-            1. Set |output|.{{MLOperand/[[operand]]}} to |outputImpl|.
-        1. Connect |input|.{{MLOperand/[[operand]]}} as input to |opImpl|.
-        1. Connect |output|.{{MLOperand/[[operand]]}} as output to |opImpl|.
+        1. Let |operator| be an [=operator=] for the PreLU operation, given |slope|.
```

Comment:
BTW, "PreLU" seems odd. It stands for "Parametric ReLU". Should it be "PReLU"?

Reply:
Done